### PR TITLE
Fix reference link in json type BBE

### PR DESCRIPTION
**Pull request by SasinduDilshara**
Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/9348

### CODE REVIEW

**Review by rdulmina — DISMISSED** [review]

File: examples/json-type/json_type.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The `json` type can be explained as a union of the simple basic types, `string`,
 
 ## Related links
 - [Access JSON elements](/learn/by-example/access-json-elements/)
-- [Converting from JSON to user defined type with langlib functions](/learn/by-example/converting-from-json-to-user-defined-type-with-langlib-functions/)
-- [Converting from user-defined type to JSON](/learn/by-example/converting-from-user-defined-type-to-json/)
+- [Convert from JSON to user-defined type](/learn/by-example/convert-from-json-to-user-defined-type/)
+- [Convert from user-defined type to JSON](/learn/by-example/converting-from-user-defined-type-to-json/)
```
Comment (LakshanWeerasinghe) [inline examples/json-type/json_type.md:14]:
```suggestion
- [Convert from user-defined type to JSON](/learn/by-example/convert-from-user-defined-type-to-json/)
```

Comment (LakshanWeerasinghe) [reply examples/json-type/json_type.md:14]: Don't we need to update the folder names as well?

Comment (SasinduDilshara) [reply examples/json-type/json_type.md:14]: We need to handle it in a separate issue.
It is not related to this bbe link failure.

**Review by MaryamZi — APPROVED** [review]
Please send the PR to the 2201.10.x branch also.

File: examples/json-type/json_type.md (hunk 1 above)
Comment (LakshanWeerasinghe) [reply examples/json-type/json_type.md:14]: Will this work if we didn't update the folder name?

Comment (SasinduDilshara) [reply examples/json-type/json_type.md:14]: yes. Folder is in the corect name

**Pull request merged**